go.mod

```diff
@@ -3,110 +3,127 @@ module github.com/virtual-kubelet/azure-aci
 go 1.18
 
 require (
-	contrib.go.opencensus.io/exporter/ocagent v0.4.12
-	github.com/Azure/azure-sdk-for-go v63.0.0+incompatible
+	contrib.go.opencensus.io/exporter/ocagent v0.7.0
 	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.1.4
 	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.2.0
 	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/containerinstance/armcontainerinstance/v2 v2.2.0-beta.1
 	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/network/armnetwork/v2 v2.1.0
-	github.com/Azure/go-autorest/autorest v0.11.24
+	github.com/Azure/go-autorest/autorest v0.11.27
 	github.com/Azure/go-autorest/autorest/adal v0.9.20
-	github.com/Azure/go-autorest/autorest/azure/auth v0.5.11
-	github.com/Azure/go-autorest/autorest/mocks v0.4.2
 	github.com/BurntSushi/toml v0.3.1
+	github.com/cpuguy83/dockercfg v0.3.1
 	github.com/dimchansky/utfbom v1.1.1
 	github.com/golang/mock v1.6.0
 	github.com/google/uuid v1.1.2
-	github.com/gorilla/mux v1.7.3
 	github.com/gorilla/websocket v1.4.2
-	github.com/onsi/gomega v1.16.0
+	github.com/mitchellh/go-homedir v1.1.0
 	github.com/patrickmn/go-cache v2.1.0+incompatible
 	github.com/pkg/errors v0.9.1
-	github.com/sirupsen/logrus v1.6.0
-	github.com/stretchr/testify v1.7.0
-	github.com/thoas/go-funk v0.9.1
-	github.com/virtual-kubelet/node-cli v0.8.0
-	github.com/virtual-kubelet/virtual-kubelet v1.6.0
-	go.opencensus.io v0.22.3
-	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
+	github.com/sirupsen/logrus v1.9.0
+	github.com/spf13/cobra v1.5.0
+	github.com/stretchr/testify v1.8.0
+	github.com/virtual-kubelet/virtual-kubelet v1.8.0
+	go.opencensus.io v0.23.0
+	golang.org/x/sync v0.0.0-20220722155255-886fb9371eb4
 	gotest.tools v2.2.0+incompatible
-	k8s.io/api v0.21.0
-	k8s.io/apimachinery v0.21.0
-	k8s.io/client-go v0.21.0
+	k8s.io/api v0.26.2
```

Comment:
Is api version 0.26.2 matching with client go version 0.25.0?

Reply:
It seems dependencies in VK have been updated to 0.26.0, updated here as well.